docs/technical-overview.md

```diff
@@ -0,0 +1,241 @@
+# Podium Request Portal — Technical Documentation
+
+## Project summary
+
+Podium Request Portal is an open source web-based application with which researchers can place requests for samples or data (including images) from Dutch biobanks and registries, and where these requests can be tracked and traced. The focus of this project is on supporting only essential (generic) steps of these request processes, keeping in mind that additional functionality might need to be added later on.
+
+Project source code is available in the following repositories:
+- https://github.com/thehyve/podium-registry — configuration registry and system health monitoring webapp;
+- https://github.com/thehyve/podium — core functionality.
+
+Project codebase is maintained by The Hyve.
+For technical questions you can contact alexey@thehyve.nl or gijs@thehyve.nl
+
+
+## Purpose of the document
+
+This document is intended to be a succinct technical reference that facilitates updating, testing and further development of the Podium project, and is aimed at people who plan to contribute to the project codebase.
+High-level architecture overview, component interactions and major dependencies are included, while implementation details that can be trivially derived from the code (e.g. DTOs) and/or autogenerated (e.g. Swagger API definition, database schema), and user-oriented nuances (e.g. visual elements of the web UIs, installation manual) are omitted.
+
+
+## Principal components
+
+### Overview
+
+
+```plantuml
+@startuml
+node "Registry" {
+  [Registry API]
+  [Admin Webapp] --> [Registry API]
+}
+
+node "UAA" {
+  [User Auth API] --> [Registry API]
+}
+
+node "Gateway" {
+  [Gateway API] --> [Registry API]
+  [Podium Webapp] --> [Gateway API]
+}
+
+cloud {
+  [Github Config]
```

Comment:
While it is possible to use a git repo, the preferred way is to pass a config file, see https://github.com/thehyve/podium/blob/dev/docs/installation.md#configure-the-registry-repository